### ISSUE TITLE
Colorbox + video + (Full size) links to nonexistant file

### ISSUE THREAD

**Issue by dsjoho** [saimn/sigal#111]
I just tried generating an album with the colorbox theme, and the album contains .MOV files. My settings include soft-linking the original material, and as such there's a (Full size) link at the top of the converted video.
However, something goes wrong as the http-links point to
PATH/original/MVI_3575.MOV
where the symbolic link created by sigal, pointing to the original file, is named
PATH/original/MVI_3575.webm
which thus leads to a 404.
However, the symbolic link is actually just a link to the .MOV, so the extension of the symbolic link should really be .MOV instead, so as to not confuse any player.

TL;DR:
Sigal creates symbolic links named <file>.webm, when the correct extension is <file>.MOV (or other format, supposedly).

this is on version 0.8.0.
